drivers/sensor/bq274xx/bq274xx.h

```diff
@@ -60,15 +60,15 @@
 #define BQ274XX_CTRL_EXIT_RESIM      0x0044
 
 /*** Extended Data Commands ***/
-#define BQ274XX_EXT_OPCONFIG                   0x3A /* OpConfig() */
-#define BQ274XX_EXT_CAPACITY                   0x3C /* DesignCapacity() */
-#define BQ274XX_EXT_DATA_CLASS                 0x3E /* DataClass() */
-#define BQ274XX_EXT_DATA_BLOCK                 0x3F /* DataBlock() */
-#define BQ274XX_EXT_BLKDAT_START               0x40 /* BlockData_start() */
-#define BQ274XX_EXT_BLKDAT_END                 0x5F /* BlockData_end() */
-#define BQ274XX_EXT_CHECKSUM                   0x60 /* BlockDataCheckSum() */
-#define BQ274XX_EXT_DATA_CONTROL               0x61 /* BlockDataControl() */
-#define BQ274XX_EXT_BLKDAT(off)                (BQ274XX_EXT_BLKDAT_START + off)
+#define BQ274XX_EXT_OPCONFIG     0x3A /* OpConfig() */
+#define BQ274XX_EXT_CAPACITY     0x3C /* DesignCapacity() */
+#define BQ274XX_EXT_DATA_CLASS   0x3E /* DataClass() */
+#define BQ274XX_EXT_DATA_BLOCK   0x3F /* DataBlock() */
+#define BQ274XX_EXT_BLKDAT_START 0x40 /* BlockData_start() */
+#define BQ274XX_EXT_BLKDAT_END   0x5F /* BlockData_end() */
+#define BQ274XX_EXT_CHECKSUM     0x60 /* BlockDataCheckSum() */
+#define BQ274XX_EXT_DATA_CONTROL 0x61 /* BlockDataControl() */
+#define BQ274XX_EXT_BLKDAT(off)  (BQ274XX_EXT_BLKDAT_START + off)
```

Comment:
There are a lot of whitespace changes unrelated to the chemistry changes. Please revert the whitespace changes or move them to a separate commit

Reply:
Or not do them at all, I could understand if you were about to do a major driver rework and maintain it but just for the sake of it does not help much. I understand you probably ran clang-format or something but then other contributors don't use it, it'll just go back out of sync again anyway.

Reply:
Fair enough, i reverted them. I ran clang-format to ensure i was following the style guide, but i understand it made the diff harder to read.